clients/include/testing_matmul.hpp

```diff
@@ -3254,14 +3268,22 @@ void testing_matmul_with_bias(const Arguments& arg,
 
         if(heuristicResult.size() > 1)
         {
-            int32_t     solutionIndex = -1;
-            std::string solutionName  = "";
-            std::string kernelName    = "";
+            int32_t solutionIndex = hipblaslt_ext::getIndexFromAlgo(heuristicResult[best_sol].algo);
```

Comment:
call getIndexFromAlgo only when it is required.

Reply:
Thanks, it’s done in commit [e10053c](https://github.com/ROCm/hipBLASLt/pull/1289/commits/e10053c53b139676cfb591224b0004baa141bd2b)